src/XRP/network-clients/json-rpc-network-client.ts

```diff
@@ -0,0 +1,45 @@
+import axios, { AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios'
+
+interface JsonRequest {
```

Comment:
can you doc this? a more descriptive name may also be helpful here (`JsonRequest` is a fairly common action on the interwebz)

Reply:
yep, in subsequent PR I move this to another file with additional JSON interface definitions. doc'ed and renamed here for now